### PR TITLE
s2u-47: New feature -> Recordings

### PR DESCRIPTION
**Pull request by frasese**
* i18n Updates
* Bug fixing

### CODE REVIEW

**Comment by mpellicer** [comment]
Does this require conversion scripts?

**Comment by frasese** [comment]
No conversion scripts are required. All recording related data is fetched directly from Microsoft and stored in cache.

File: meetings/tool/src/main/resources/card _eu.properties [hunk 1]
```diff
@@ -1,18 +1,19 @@
-decription_link_text=View Description
```
Comment (jesusmmp) [inline meetings/tool/src/main/resources/card _eu.properties:1]: Name file should be card_eu.properties. There is a blank space before _

**Comment by jesusmmp** [comment]
Awesome work!

File: meetings/tool/src/main/resources/meeting-recordings_eu.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+back=Back
```
Comment (mpellicer) [inline meetings/tool/src/main/resources/meeting-recordings_eu.properties:1]: @pablosanroman Could you provide these strings when you have time? Thanks

File: meetings/tool/src/main/resources/card _eu.properties (hunk 1 above)
Comment (mpellicer) [reply meetings/tool/src/main/resources/card _eu.properties:1]: I think he fixed it!

**Comment by mpellicer** [comment]
THIS IS AWESOME @frasese , you're a monster!

File: meetings/tool/src/main/resources/meeting-recordings_eu.properties (hunk 2 above)
Comment (PabloSanRoman) [reply meetings/tool/src/main/resources/meeting-recordings_eu.properties:1]: Hi @mpellicer !

empty_field= Eremu hau nahitaez bete behar da
empty_checkbox= Kontrol-lauki hau nahitaez aktibatu behar da
maxlenght_field= Balioak gainditzen du baimendutako gehienezko tamaina

Salu:D!

**Pull request merged**